modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -57,9 +61,24 @@ export default {
     progress: {
       type: Object,
       default: null
+    },
+    dismiss: {
+      type: Number,
+      default: 0
+    },
+    pulse: {
```

Comment:
What is this used for?

Reply:
Pulse? I don't know, it was in 2.x. Shall I remove it?

Reply:
Yes, I think so.

Reply:
Let's remove this.

Reply:
Done